test/lit/wat-kitchen-sink.wast

```diff
@@ -463,6 +463,11 @@
  ;; CHECK:      (start $return-none)
  (start $return-none)
 
+ ;; Annotations
+ (@annotation this is a meaningless (@annotation ) ;; This is still a comment ))
+   it spans multiple lines just fine and can include $ids 0x42 numbers and "strings"
+ )
```

Comment:
Should this not have a `CHECK`?

Reply:
Nope, the parser never does anything with it, so it's essentially a comment.

Reply:
Doesn't it need to roundtrip it?

Reply:
No. There is a specific kind of annotation (`(@custom ...)`) that specifies arbitrary custom section contents that should be round-tripped, but we don't support that yet.

Reply:
I see, thanks, that's what I was missing.